devlab/tests/testcases/test_verify_dst_functionality.py

```diff
@@ -23,10 +23,12 @@
 
 import devlab.tests.config as config
 import devlab.tests.functional_test as functional_test
+from devlab.tests.base import BasePrerequisites as base
```

Comment:
If it possible, let's start using the general openstack rules: http://docs.openstack.org/developer/hacking/#imports